meta-xt-prod-cockpit-rcar-driver-domain/recipes-core/images/core-image-minimal.bbappend

```diff
@@ -13,8 +13,23 @@ IMAGE_INSTALL_append = " \
     optee-os \
     fio \
     iotop \
+    cluster-service \
+    aos-vis \
```

Comment:
I'm not sure that we need AOS-VIS in DomD. As I understand, AOS-VIS server is located on separate board.
So, we need VisClient but not AOS-VIS server.